### ISSUE TITLE
Bug: Client - The distinct seed for order creation is is not thread-friendly 

### ISSUE THREAD

**Issue by druspencer** [MonacoProtocol/protocol#83]
Currently, when we generate a PDA for an order, we apply a distinct seed, this seed is based on the time the PDA is generated:

```
const distinctSeed = Date.now().toString();
```

When performing threaded operations on the same market for a single wallet, it is possible to encounter a PDA collision, with the resulting order creation for one of the orders returning:

```
"Error: failed to send transaction: Transaction simulation failed: Error processing Instruction 2: custom program error: 0x0"
```

This means the account you are trying to initialize has already been initialized (as the PDA is the same). It would be expected that the distinct seed is unique every time a PDA is generated in order to allow for thread-safe operations for client consumers.

**Comment by druspencer** [comment]
This issue is currently being worked on 👍

**Comment by druspencer** [comment]
UUID work has been merged into develop and will be released in the client alongside 0.12.0 https://github.com/MonacoProtocol/protocol/pull/125
